<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dell-i37-9Z1F2P2-w11x64" failures="1" tests="5" name="Test" time="0.393" errors="0" timestamp="2025-06-25T21:06:14 IST">
  <testcase classname="ReRun4_Failed_TCs_using_TestNgFailedXMLfile.S2_ClassB" name="testCase3" time="0.066"/>
  <testcase classname="ReRun4_Failed_TCs_using_TestNgFailedXMLfile.S2_ClassB" name="testCase4" time="0.011">
    <skipped/>
  </testcase> <!-- testCase4 -->
  <testcase classname="ReRun4_Failed_TCs_using_TestNgFailedXMLfile.S2_ClassB" name="testCase4" time="0.003">
    <skipped/>
  </testcase> <!-- testCase4 -->
  <testcase classname="ReRun4_Failed_TCs_using_TestNgFailedXMLfile.S2_ClassB" name="testCase4" time="0.026">
    <skipped/>
  </testcase> <!-- testCase4 -->
  <testcase classname="ReRun4_Failed_TCs_using_TestNgFailedXMLfile.S2_ClassB" name="testCase4" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ReRun4_Failed_TCs_using_TestNgFailedXMLfile.S2_ClassB.testCase4(S2_ClassB.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testCase4 -->
</testsuite> <!-- Test -->
